libvast/src/concept/parseable/vast/expression.cpp

```diff
@@ -56,13 +56,69 @@ static predicate::operand to_data_operand(data x) {
   return x;
 }
 
-static predicate to_value_predicate(data x) {
+// Expands a predicate with a type extractor, an equality operator, and a
+// corresponding data instance according to the rules of the expression
+// language.
```

Comment:
Is doxygenize a word? If so, please doxygenize the comment by adding another `/`.

Reply:
Disagreeing here; this is an implementation detail local to the function. It will never land in documentation.

Reply:
Tooling disagrees with you. I get a different hint from clangd on autocomplete with three slashes.

Reply:
Thank god I'm a human and can make decisions with more context. 😉

Reply:
Joking aside, this visitor is in an anonymous namespace. It should not be publicly documented.

Reply:
Per our call, turns out the de-facto standard for language servers is to use triple slashes...who would have known.